crates/starknet-server/src/cli.rs

```diff
@@ -1,33 +1,69 @@
 use clap::Parser;
 use starknet_core::StarknetConfig;
+use starknet_in_rust::definitions::block_context::StarknetChainId;
 use starknet_types::num_bigint::BigUint;
 
 /// Run a local instance of Starknet Devnet
 #[derive(Parser, Debug)]
 #[command(author, version, about, long_about = None)]
 #[command(propagate_version = true)]
 pub(crate) struct Args {
-    /// Number of accounts
+    /// Number of accounts.
     #[arg(long = "accounts")]
     #[arg(value_name = "ACCOUNTS")]
     #[arg(default_value = "10")]
     #[arg(help = "Specify the number of accounts to be predeployed;")]
     accounts_count: u8,
 
-    /// initial balance of predeployed accounts
+    /// Initial balance of predeployed accounts.
     #[arg(long = "initial-balance")]
     #[arg(short = 'e')]
     #[arg(value_name = "INITIAL_BALANCE")]
     #[arg(default_value = "1000000000000000000000")]
     #[arg(help = "Specify the initial balance in WEI of accounts to be predeployed;")]
     initial_balance: BigUint,
 
-    // seed for predeployed accounts
+    // Seed for predeployed accounts.
     #[arg(long = "seed")]
     #[arg(value_name = "SEED")]
     #[arg(help = "Specify the seed for randomness of accounts to be predeployed; if not \
                   provided, it is randomly generated")]
     seed: Option<u32>,
+
+    // Host address.
+    #[arg(long = "host")]
+    #[arg(value_name = "HOST")]
+    #[arg(default_value = "127.0.0.1")]
+    #[arg(help = "Specify the address to listen at;")]
+    host: String,
+
+    // Port number.
+    #[arg(long = "port")]
+    #[arg(value_name = "PORT")]
+    #[arg(default_value = "5050")]
```

Comment:
`default_value_t` shall be used for data types.
https://github.com/clap-rs/clap/discussions/3640